src/Microsoft.VisualBasic.Core/src/Microsoft/VisualBasic/Interaction.vb

```diff
@@ -26,6 +129,23 @@ Namespace Microsoft.VisualBasic
 #End If
         End Sub
 
+        Public Function InputBox(ByVal Prompt As String, Optional ByVal Title As String = "", Optional ByVal DefaultResponse As String = "", Optional ByVal XPos As Integer = -1, Optional ByVal YPos As Integer = -1) As String
+            Return DirectCast(InvokeMethod("InputBox", Prompt, Title, DefaultResponse, XPos, YPos), String)
+        End Function
+
+        Public Function MsgBox(ByVal Prompt As Object, Optional ByVal Buttons As MsgBoxStyle = MsgBoxStyle.OkOnly, Optional ByVal Title As Object = Nothing) As MsgBoxResult
+            Return DirectCast(InvokeMethod("MsgBox", Prompt, Buttons, Title), MsgBoxResult)
+        End Function
+
+        Private Function InvokeMethod(methodName As String, ParamArray args As Object()) As Object
+            Dim type As Type = type.GetType("Microsoft.VisualBasic._Interaction, Microsoft.VisualBasic, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a", throwOnError:=False)
+            Dim method As MethodInfo = type?.GetMethod(methodName)
+            If method Is Nothing Then
+                Throw New PlatformNotSupportedException()
```

Comment:
You may want an error message here that indicates what might be done to fix the problem.